gensrc/thrift/PaloInternalService.thrift

```diff
@@ -274,8 +274,18 @@ struct TTxnParams {
   10: optional double max_filter_ratio
 }
 
-// ExecPlanFragment
+// Definition of global dict, global dict is used to accelerate query performance of low cardinality data
+struct TColumnDict {
+  1: optional Types.TPrimitiveType type
+  2: list<string> str_dict  // map one string to a integer, using offset as id
+}
 
+struct TGlobalDict {
+  1: optional map<i32, TColumnDict> dicts,  // map dict_id to column dict
```

Comment:
Do we need a type as big as i32 here?

Reply:
Maybe need. Some times I find there are sql queries that have more than 10k lines, I am afraid there are many dicts. And the slot id has int32 type.